application/contract/rootdomain/util/util.go

```diff
@@ -0,0 +1,30 @@
+package util
```

Comment:
are you sure you need a separate package for just 3 functions? better to move them in util.go file to package there they actually needed or move to proper packages according to they purpose